### DISCUSSION TITLE
Recreate cartoon drawing style

### DISCUSSION THREAD

**Discussion by quasar136a8** [lllyasviel/Fooocus#2875] [Show and tell]
Hi, can someone help me recreate some of these images with the same style (character). I can’t find the required style, or is it more of a quality prompt?
![Pinterest (1)](https://github.com/lllyasviel/Fooocus/assets/111929719/ad414c8c-a2cb-40ec-9547-5d7a7e4106aa)
![Pinterest (2)](https://github.com/lllyasviel/Fooocus/assets/111929719/05f413e1-c3a1-4fcd-88c3-d669aab800f8)
![Screenshot - 2024-05-06 10 26 47 (1)](https://github.com/lllyasviel/Fooocus/assets/111929719/1049e810-0b08-4ed5-bf60-37978e3efa45)

**Comment by Rogala** [comment]
[Didn't see anything like that, only this came from memory](https://civitai.com/models/265129/cute-drawing)

**Reply by quasar136a8** [reply]
<img width="406" alt="image" src="https://github.com/lllyasviel/Fooocus/assets/111929719/ba9fd83f-a35b-4907-a468-507b321bb713">
What to choose here?

**Reply by Rogala** [reply]
You don't have to specify a style, just play with Lora weight.

[Try some of these styles to see if anything fits](https://github.com/lllyasviel/Fooocus/discussions/2685)
